repositories/UserRepository.ts

```diff
@@ -91,4 +91,12 @@ export class UserRepository extends BaseRepository<UserModel> {
       })
       .execute();
   }
+
+  public async getAllProfilesForRoleManagement() {
+    const profiles = await this.repository
+      .createQueryBuilder()
+      .select(['uuid', 'handle', 'email', 'UserModel.firstName', 'UserModel.lastName', 'UserModel.accessType'])
+      .getRawMany();
+    return profiles;
+  }
```

Comment:
nit: I don't think this function name should describe what it is used for. e.g., if we want to use this functionality for something other than role management, then it would need to be renamed. Something like `getUserInfoAndAccessTypes()` might suffice.